src/Test.FunctionalTests.BinSkim.Driver/AnalyzeCommandTest.cs

```diff
@@ -1,15 +1,77 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.BinaryParsers;
+using Microsoft.CodeAnalysis.IL;
+using Microsoft.CodeAnalysis.Sarif;
+
+using Moq;
+
+using Newtonsoft.Json;
+
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.BinSkim.Driver
 {
     public class AnalyzeCommandTest
     {
+        private const string ToolName = "binskim";
+        private readonly string SarifLogV1 = $@"{{
```

Comment:
>SarifLogV1 [](http://example.com/codeflow?start=32&length=10)

Can't we create this object dynamically?

Reply:
we will check-in a new file with this content.